src/file/mod.rs

```diff
@@ -8,9 +8,6 @@ use std::ffi::OsStr;
 use std::fs;
 use std::path::{Path, PathBuf};
 
-/* Extension of the configuration file */
-const CONFIG_FILE_EXTENSION: &str = "conf";
```

Comment:
Can we keep this constant? 👀

Reply:
@orhun This constant is only used once, thus I delete this :)